attributes/default.rb

```diff
@@ -306,6 +288,38 @@
   end
 end
 
+# Lustre defaults (for CentOS >=7.7 and Ubuntu)
+default['cfncluster']['lustre']['public_key'] = value_for_platform(
+  'centos' => { '>=7.7' => "https://fsx-lustre-client-repo-public-keys.s3.amazonaws.com/fsx-rpm-public-key.asc" },
+  'ubuntu' => { :default => "https://fsx-lustre-client-repo-public-keys.s3.amazonaws.com/fsx-ubuntu-public-key.asc" }
```

Comment:
Nice use of `value_for_platform `!
Just curious, is `:default` the same as `'default'`? From [docs here](https://docs.chef.io/dsl_recipe/#value_for_platform) I can't seem to find `:default` syntax

Reply:
`:default` or `default:` are symbols. Using symbol is an [alternative way](https://launchschool.com/books/ruby/read/hashes) to define keys in hashes. See also [symbols vs strings](https://medium.com/@lcriswell/ruby-symbols-vs-strings-248842529fd9).

In this specific case I started using the symbol because Intellij was showing me a warning: `"String key in hash" --> "Convert to symbol key"`.

The bad is that `:default` is not accepted by rubocop (we're following [ruby19 syntax](https://www.rubydoc.info/gems/rubocop/0.46.0/RuboCop/Cop/Style/HashSyntax)) and mixing the two syntaxes is not a clear solution for me, so for now I'm going use the same syntax for all the keys, despite to IntelliJ warning.

See also: https://docs-archive.chef.io/release/11-18/ruby.html#symbols-or-strings